chat/run-tests.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Color output for better visibility
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+cd /tests
+
+echo "Running tests in ${TEST_ENVIRONMENT}"
+
+nx run chat:test:acc 
```

Comment:
What happens if we just do `nx run test:acc` will it just run whichever domain it's within (i.e. the chat domain because we've copied only chat files into the dockerfile)?